dbt/include/teradata/macros/materializations/incremental/strategies.sql

```diff
@@ -0,0 +1,25 @@
+
+{% macro teradata__get_incremental_default_sql(arg_dict) %}
+
+    {% do return(get_incremental_append_sql(arg_dict)) %}
+
+{% endmacro %}
+
+
+{% macro teradata__get_incremental_append_sql(target_relation, tmp_relation,dest_columns) %}
+
+  {% do return(get_insert_into_sql(target_relation, tmp_relation,  dest_columns)) %}
+
+{% endmacro %}
+
+
+{% macro get_insert_into_sql(target_relation, tmp_relation, dest_columns) %}
+
+    {%- set dest_cols_csv = get_quoted_csv(dest_columns | map(attribute="name")) -%}

```

Comment:
Is this statement different from {%- set dest_cols_csv = dest_columns | map(attribute='quoted') | join(', ') -%}?

Reply:
I think it's just the other way of writing the same thing.